CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Runner Manager Contribution and Governance Policies
+This document describes the contribution process and governance policies of the Runner Manager project. The project is also governed by QAlity tech.
+
+# Contribution Process
+Before making a contribution, please take the following steps:
+
+- Check whether there's already an open issue related to your proposed contribution. If there is, join the discussion and propose your contribution there.
+- If there isn't already a relevant issue, create one, describing your contribution and the problem you're trying to solve.
+- Respond to any questions or suggestions raised in the issue by other developers.
+- Fork the project repository and prepare your proposed contribution.
+- Submit a pull request.
+
+NOTE: All contributors must have a contributor license agreement (CLA) on file with before their pull requests will be merged. Please review the contribution requirements and submit (or have your employer submit) the required CLA before submitting a pull request.
```

Comment:
on file with us?

Reply:
according to https://en.wikipedia.org/wiki/Contributor_License_Agreement it is not a bad idea. However, we can remove that line and go for https://docs.fedoraproject.org/en-US/legal/fpca/ as the WIKI suggested.

Reply:
Huh? I meant there's a typo, it should be "on file with us", it's currently "on file with" 😄